Shift3/boilerplate-client-react
# Added `admin route guards`

## Changes
1. Updated `PrivateRoute.tsx` to allow for a 'Role Type' check.
2. Moved the router to Index.tsx.

## Purpose
To add `Admin` route guards based on role type.

## Testing
1. Pull in the changes to your local copy of this branch and run `yarn start` and make sure the app loads correctly.
2. Check if you are able to navigate to various routes based on your role type.
